HousingSearchApi/data/elasticsearch/index.json

```diff
@@ -0,0 +1,215 @@
+{
+  "settings": {
+    "analysis": {
+      "analyzer": {
+        "whitespace_removed": {
+          "tokenizer": "keyword",
+          "filter": [
+            "lowercase"
+          ],
+          "char_filter": [
+            "remove_whitespace"
+          ]
+        },
+        "address_text": {
+          "type": "custom",
+          "tokenizer": "classic",
+          "ignore_case": true,
+          "char_filter": [
+            "remove_apostrophes"
+          ],
+          "filter": [
+            "lowercase",
+            "common_street_terms_synonyms",
+            "ignore_unmeaningful_building_words",
+            "remove_numbers"
+          ]
+        },
+        "extract_number_analyzer" : {
+          "tokenizer" : "standard",
+          "filter" : ["extract_numbers", "decimal_digit"]
+        }
+      },
+      "char_filter": {
+        "remove_whitespace": {
+          "type": "pattern_replace",
+          "pattern": " ",
+          "replacement": ""
+        },
+        "remove_apostrophes": {
+          "type": "pattern_replace",
+          "pattern": "'",
+          "replacement": ""
+        }
+      },
+      "filter": {
+        "common_street_terms_synonyms": {
+          "type": "synonym",
+          "lenient": false,
+          "synonyms": [ "road, raod, lane, lanes, street, streat, avenue, rd, str, avn, ave, close => road" ]
+        },
+        "ignore_unmeaningful_building_words": {
+          "type": "stop",
+          "ignore_case": true,
+          "stopwords": [
+            "flat",
+            "flt",
+            "apt",
+            "apartment",
+            "fat"
+          ]
+        },
+        "extract_numbers" : {
+          "type" : "keep_types",
+          "types" : [ "<NUM>" ]
+        },
+        "remove_numbers" : {
+          "type" : "keep_types",
+          "types" : [ "<NUM>" ],
+          "mode": "exclude"
+        }
+      }
+    }
+  },
+  "mappings" : {
```

Comment:
I've never done ES before
Why are the mappings linking to asset (properties) should it not be linking to persons?

Reply:
This will change as I start to work on the actual ticket for ES :) don't worry.